components/mpas-ocean/src/Registry.xml

```diff
@@ -343,6 +343,28 @@
 					possible_values="Any positive real value."
 		/>
 	</nml_record>
+	<nml_record name="submesoscale_eddy_parameterization" mode="forward">
+	    <nml_option name="config_submesoscale_enable" type="logical" default_value=".false." units="NA"
+		            description="flag to enable the FK2011 parameterization for submesoscale eddies"
+					possible_values=".true. or .false."
+					/>
+	    <nml_option name="config_submesoscale_tau" type="real" default_value="172800" units="s"
+		            description="timescale for frictional slumping of front (in seconds)"
+					possible_values="positive reals, between 1-10 days"
+					/>
+        <nml_option name="config_submesoscale_Ce" type="real" default_value="0.06" units="NA"
```

Comment:
I don't understand why these show up with different spacing.

Reply:
Possibly a tabs vs spaces issue.

Reply:
yep you were exactly right.  I think I have it fixed.